lib/rmt/cli/products.rb

```diff
@@ -23,18 +22,40 @@ def list
       else
         warn _('No matching products found in the database.')
       end
+    elsif options.csv
+      data = products.map do |product|
+        [
+          product.id,
+          product.shortname,
+          product.version,
+          product.arch,
+          product.product_string,
+          product.mirror?,
```

Comment:
Is there a specific reason why we're omitting information about the release stage here? Or is it just for consistency reasons?